include/sys/__assert.h

```diff
@@ -8,7 +8,7 @@
 #define ZEPHYR_INCLUDE_SYS___ASSERT_H_
 
 #include <stdbool.h>
-#include <sys/printk.h>
+#include <toolchain.h>
```

Comment:
What requires this header in this file?

Reply:
Z_STRINGIFY

Reply:
I thought that was part of the `sys/util.h` -- it probably should be, but apparently not.